service-api/src/main/java/greencity/service/CommentService.java

```diff
@@ -105,4 +114,13 @@ AddCommentDtoResponse save(ArticleType articleType, Long articleId, AddCommentDt
      * @param userVO current {@link CommentVO} that wants to delete.
      */
     void delete(Long id, UserVO userVO);
+
+    /**
+     * Method that allow you to search users by name.
+     *
+     * @param searchUsers dto with current user ID and search query
+     *                    {@link UserSearchDto}.
+     * @author Anton Bondar
+     */
+    void searchUsers(UserSearchDto searchUsers);
```

Comment:
Why is `search` method void?
I suppose the `searchUsers` method should return list of users

Reply:
Because we are sending a message with the search results to a messaging system (WebSocket).So, the method does not return any value directly to the caller of searchUsers.